src/utils/asset.ts

```diff
@@ -0,0 +1,19 @@
+import { AssetAvailability, DataObject } from '@/api/queries'
+import { STORAGE_NODE_URL } from '@/config/urls'
+
+export const createUrlFromAsset = (
+  availability?: AssetAvailability,
+  assetUrl?: string | null,
+  dataObject?: DataObject | null
+) => {
+  if (availability !== AssetAvailability.Accepted) {
+    return
```

Comment:
maybe we add a `console.warn` here? I don't suppose we should run into a situation in which we try to display an unavailable asset.
```suggestion
    console.warn('Trying to create URL for non-accepted asset')
    return
```